Documentation/building/osx-instructions.md

```diff
@@ -139,69 +142,88 @@ The rest of the assemblies you need to run are presently just facades that point
-    <packages>
-      <package id="System.Console" version="4.0.0-beta-22703" />
-      <package id="System.Diagnostics.Contracts" version="4.0.0-beta-22703" />
-      <package id="System.Diagnostics.Debug" version="4.0.10-beta-22703" />
-      <package id="System.Diagnostics.Tools" version="4.0.0-beta-22703" />
-      <package id="System.Globalization" version="4.0.10-beta-22703" />
-      <package id="System.IO" version="4.0.10-beta-22703" />
-      <package id="System.IO.FileSystem.Primitives" version="4.0.0-beta-22703" />
-      <package id="System.Reflection" version="4.0.10-beta-22703" />
-      <package id="System.Resources.ResourceManager" version="4.0.0-beta-22703" />
-      <package id="System.Runtime" version="4.0.20-beta-22703" />
-      <package id="System.Runtime.Extensions" version="4.0.10-beta-22703" />
-      <package id="System.Runtime.Handles" version="4.0.0-beta-22703" />
-      <package id="System.Runtime.InteropServices" version="4.0.20-beta-22703" />
-      <package id="System.Text.Encoding" version="4.0.10-beta-22703" />
-      <package id="System.Text.Encoding.Extensions" version="4.0.10-beta-22703" />
-      <package id="System.Threading" version="4.0.10-beta-22703" />
-      <package id="System.Threading.Tasks" version="4.0.10-beta-22703" />
-    </packages>
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Console" version="4.0.0-beta-22703" />
+  <package id="System.Diagnostics.Contracts" version="4.0.0-beta-22703" />
+  <package id="System.Diagnostics.Debug" version="4.0.10-beta-22703" />
+  <package id="System.Diagnostics.Tools" version="4.0.0-beta-22703" />
+  <package id="System.Globalization" version="4.0.10-beta-22703" />
+  <package id="System.IO" version="4.0.10-beta-22703" />
+  <package id="System.IO.FileSystem.Primitives" version="4.0.0-beta-22703" />
+  <package id="System.Reflection" version="4.0.10-beta-22703" />
+  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22703" />
+  <package id="System.Runtime" version="4.0.20-beta-22703" />
+  <package id="System.Runtime.Extensions" version="4.0.10-beta-22703" />
+  <package id="System.Runtime.Handles" version="4.0.0-beta-22703" />
+  <package id="System.Runtime.InteropServices" version="4.0.20-beta-22703" />
+  <package id="System.Text.Encoding" version="4.0.10-beta-22703" />
+  <package id="System.Text.Encoding.Extensions" version="4.0.10-beta-22703" />
+  <package id="System.Threading" version="4.0.10-beta-22703" />
+  <package id="System.Threading.Tasks" version="4.0.10-beta-22703" />
+</packages>
+```
 
 And restore your packages.config file:
 
-    dotnet-mbp:~ richlander$ cd ~/coreclr-demo
-    dotnet-mbp:coreclr-demo richlander$ mono nuget.exe restore packages/packages.config -Source https://www.myget.org/F/dotnet-corefx/ -PackagesDirectory packages
+```sh
+dotnet-mbp:~ richlander$ cd ~/coreclr-demo
+dotnet-mbp:coreclr-demo richlander$ mono nuget.exe restore packages/packages.config \
```

Comment:
Thanks. I have aligned the docs with Linux, removing all the redundant instructions about nuget, mono and running the demo based on the conversation in PR https://github.com/dotnet/coreclr/pull/3477.